packages/functional-tests/playwright.config.ts

```diff
@@ -28,6 +28,8 @@ export default defineConfig<PlaywrightTestConfig<TestOptions, WorkerOptions>>({
   forbidOnly: CI,
   retries,
   testDir: 'tests',
+  // Run all tests in parallel.
+  fullyParallel: true,
```

Comment:
Bonus points if we bind this to a CLI flagged so we can just use `YOLO=true`.
I'm actually really interested in your report on this.

Hypothesis: Slightly faster runs due to generally increased parallelization, but same overall speeds on the shards because of increased retries?

Reply:
I have to look more closely at the xml reports to confirm, but I believe this allows the runner to distribute tests more evenly across workers on a shard...

Reply:
I'll try reviewing the Junit reports (and/or maybe Playwright reporters) and see if there is anything that gives us the retry test count metadata. Might be another very useful metric to track and possibly add to the Playwright reporter if possible:

```
Shard 0: Test suite: passed (Passed: 36 Failed: 0 Skipped: 12 (Fixme: 1))
Shard 1: Test suite: passed (Passed: 53 Failed: 0 Skipped: 11 (Fixme: 2))
Shard 2: Test suite: passed (Passed: 22 Failed: 0 Skipped: 18 (Fixme: 2))
Shard 3: Test suite: passed (Passed: 35 Failed: 0 Skipped:  6 (Fixme: 0))
Shard 4: Test suite: passed (Passed: 25 Failed: 0 Skipped: 16 (Fixme: 4))
Shard 5: Test suite: passed (Passed: 69 Failed: 0 Skipped:  5 (Fixme: 1))
```